library/src/morpheus-master/sass/modules/tool/forums/_forums.scss

```diff
@@ -105,6 +105,13 @@
 		}
 	}
 
+	div.specialLink{
+		max-width: calc(100% - 260px);
```

Comment:
This magic number is a bit of a hack. How was it calculated?

I suppose this will work as a temporary fix.

I'd prefer this to be fixed by removing the floating and the absolute positioning from the itemNav class (for all of Sakai). Maybe this can be a future SWITCH fix for Forums:

*Proposed Before:*
![forums-before](https://user-images.githubusercontent.com/12685096/49974118-db105480-ff05-11e8-893b-29155a90ef17.png)

*Proposed After:*
![forums-after](https://user-images.githubusercontent.com/12685096/49974127-e19ecc00-ff05-11e8-957a-d3866343a724.png)

Reply:
The number is simply the width of the two buttons, I believe. Also in your proposed after screenshot, the breadcrumbs wrap while there's still space in the row, leaving a big white gap.

Reply:
Actually, I purposely move the "General Discussion" heading to its own line so I could use it as a replacement for the duplicated topic title that appears in the yellow message banner. The breadcrumb path with wrap above the heading though, if necessary.

Reply:
Discussed with Shawn. Refactoring the top of all Forums pages is out of scope for this fix.